universal-login-sdk/test/observers/AuthorisationObserver.ts

```diff
@@ -76,6 +76,33 @@ describe('SDK: RelayerObserver', async () => {
     expect(callback2).to.have.been.calledTwice;
   });
 
+  it('returned authorisation objects have property: securityCodeWithFakes', async () => {
+    const callback = sinon.spy();
+    const unsubscribe = authorisationsObserver.subscribe(getAuthorisationRequest, callback);
+
+    await waitUntil(() => !!callback.firstCall);
+    expect(callback).to.be.calledOnce;
+    expect(callback).to.be.calledWith([]);
+
+    await sdk.connect(contractAddress);
+    await waitUntil(() => !!callback.secondCall);
+    unsubscribe();
+    expect(callback).to.be.calledTwice;
+    const authorisationsArray = callback.args[1][0];
+
+    for (const authorisation of authorisationsArray) {
+      expect('securityCodeWithFakes' in authorisation).to.be.true;
+    }
+
+    for (const authorisation of authorisationsArray) {
+      expect(authorisation.securityCodeWithFakes.length).to.eq(12);
+      for (const codeElement of authorisation.securityCodeWithFakes) {
+        expect(codeElement).to.be.gte(0);
+        expect(codeElement).to.be.lt(1024);
+      }
+    }
```

Comment:
`expect(authorisationsArray).to.have.lengthOf(12)`
or
`expectToBeProperSecurityCodeWithFakes()`